### PR TITLE
Emails and notifs

### PR DESCRIPTION
**Pull request by yspringfield**
Fixed: #8833
Here are those that haven't been modified because they are not implemented as of yet in the code.
https://docs.google.com/spreadsheets/d/1b1K7YfnquxnfCgGGPxIz5ZNIjr9eK_bUrRJqLvPYGIk/edit#gid=874886288&range=7:7
https://docs.google.com/spreadsheets/d/1b1K7YfnquxnfCgGGPxIz5ZNIjr9eK_bUrRJqLvPYGIk/edit#gid=874886288&range=12:12
https://docs.google.com/spreadsheets/d/1b1K7YfnquxnfCgGGPxIz5ZNIjr9eK_bUrRJqLvPYGIk/edit#gid=874886288&range=30:32
https://docs.google.com/spreadsheets/d/1b1K7YfnquxnfCgGGPxIz5ZNIjr9eK_bUrRJqLvPYGIk/edit#gid=874886288&range=34:35

### CODE REVIEW

**Review by RemcoSimonides — COMMENTED** [review]
I don't see changes related to

2/ All ADMIN emails from the Deal negotiation flow are sent from Sendgrid instead of the app and it triggers a million emails. Could we clean them and send them from the app (just as we do when there is a new user for example)?

`d-f45a08ce5be94e368f868579fa72afa8`
`d-ac1dfcbf692d4ef8888ccad9b6938c8e`
`d-6750c3d980834301a057403668deceed`
`d-fba5c684bdbc4187b27b82e2ba703a2a`

Will you do that in a new PR?

File: libs/notification/src/lib/service.ts [hunk 1]
```diff
@@ -272,6 +277,7 @@ export class NotificationService extends BlockframesCollection<Notification> {
           message,
           imgRef: notification.user.avatar,
           placeholderUrl: 'profil_user.svg',
+          actionText:'See Title',
```
Comment (phpgeekfr) [inline libs/notification/src/lib/service.ts:280]: bad formatting on the `actionText:xxx',` ;)

File: libs/notification/src/lib/service.ts [hunk 2]
```diff
@@ -415,6 +428,7 @@ export class NotificationService extends BlockframesCollection<Notification> {
           ...notification,
           _meta: { ...notification._meta, createdAt: notification._meta.createdAt },
           message,
+          actionText:"Access App",
```
Comment (phpgeekfr) [inline libs/notification/src/lib/service.ts:431]: use single quotes

File: apps/backend-functions/src/notification.ts [hunk 3]
```diff
@@ -649,15 +653,16 @@ async function sendCreatedCounterOfferConfirmation(recipient: User, notification
   const title = await getDocument<Movie>(`movies/${negotiation.titleId}`);
   const isMailRecipientBuyer = recipient.orgId === negotiation.buyerId;
   const app: App = 'catalog';
+  const from = getMailSender('catalog');
```
Comment (RemcoSimonides) [inline apps/backend-functions/src/notification.ts:656]:
```suggestion
  const from = getMailSender(app);
```

File: apps/backend-functions/src/templates/mail.ts [hunk 4]
```diff
@@ -591,4 +574,48 @@ export function eventCreatedAdminEmail(app: App, event: Event<EventMeta>) {
       Visit ${appUrl.crm}${ADMIN_REVIEW_EVENT_PATH}/${event.id} or go to ${ADMIN_REVIEW_EVENT_PATH}/${event.id} to review it.
     `
   };
-}
\ No newline at end of file
+}
+
+/** Inform Archipel Content admins a new offer has been created*/
+export function adminOfferCreatedConfirmationEmail(toUser: UserEmailData, org: OrgEmailData) {
+  console.log('building admin email')
```
Comment (RemcoSimonides) [inline apps/backend-functions/src/templates/mail.ts:581]: remove

File: apps/backend-functions/src/templates/mail.ts [hunk 5]
```diff
@@ -591,4 +574,48 @@ export function eventCreatedAdminEmail(app: App, event: Event<EventMeta>) {
...
+  return {
+    to: supportEmails.catalog,
+    subject: `${org.name} created a new Offer.`,
+    text: `
+      Date: ${format(new Date(), 'dd MMM, yyyy')}
+      Organization name: ${org.name}
+      Buyer name: ${toUser.firstName} ${toUser.lastName}
```
Comment (RemcoSimonides) [inline apps/backend-functions/src/templates/mail.ts:588]:
```suggestion
      Buyer name: ${displayName(toUser)}
```

File: libs/notification/src/lib/service.ts [hunk 6]
```diff
@@ -131,16 +134,16 @@ export class NotificationService extends BlockframesCollection<Notification> {
...
         return {
           ...notification,
           _meta: { ...notification._meta, createdAt: notification._meta.createdAt },
           message,
           imgRef: notification.user.avatar,
           placeholderUrl: 'profil_user.svg',
           url: `${applicationUrl['festival']}${module === 'marketplace'
-              ? `/event/${notification.docId}/r/i/`
-              : `/c/o/${module}/event/${notification.docId}`
+            ? `/event/${notification.docId}/r/i/`
```
Comment (RemcoSimonides) [inline libs/notification/src/lib/service.ts:145]: create this url outside of the return statement, it's quite unreadable like this

**Review by RemcoSimonides — APPROVED** [review]

**Pull request merged**